Cognite.Simulator.Utils/ModelLibraryBase.cs

```diff
@@ -28,10 +29,8 @@ public abstract class ModelLibraryBase<T, U, V> : FileLibrary<T, U>, IModelProvi
         where U : ModelStateBasePoco
         where V : ModelParsingInfo, new()
     {
-        /// <summary>
-        /// Staging area, used to store the model parsing info in CDF
-        /// </summary>
-        protected StagingArea<V> Staging { get; }
+        private readonly ILogger _logger;
+        private readonly ScopedRemoteApiSink _remoteApiSink;
```

Comment:
```suggestion
```

we decided to flush the logs on the heartbeat. no need to have the ScopedRemoteApiSink here